apps/system/style/chrome/chrome.css

```diff
@@ -497,7 +500,7 @@ html[dir="rtl"] .search-app .chrome.maximized .controls .urlbar:after {
 
 html[dir="rtl"] .controls .urlbar {
   transform-origin: top right;
-  
+
```

Comment:
Not sure - I think my editor just auto-fixed the trailing whitespace. I'll remove it.